handlers/life_journey_test.go

```diff
@@ -26,3 +29,46 @@ func TestLifeJourney(t *testing.T) {
 		assert.Equal(t, http.StatusOK, rec.Code)
 	}
 }
+
+func TestLifeJourneyBenefits(t *testing.T) {
+	// Setup Echo service
+	e := echo.New()
+
+	lifejourneys.LifeJourneyService = lifejourneys.LifeJourneyServiceStruct{Filename: "../life_journeys.json"}
+	// Setup http request using httptest
+	req := httptest.NewRequest(http.MethodGet, "/lifejourneys/1/benefits", nil)
+	// Create a httptest record
+	rec := httptest.NewRecorder()
+	// Create a new Echo Context
+	c := e.NewContext(req, rec)
+	c.SetParamNames("id")
+	c.SetParamValues("1")
+
+	// Assertions
+	if assert.NoError(t, HandlerService.LifeJourneyBenefits(c)) {
+		assert.Equal(t, http.StatusOK, rec.Code)
+	}
+}
+
+func TestLifeJourneyBenefits_One(t *testing.T) {
+	// Setup Echo service
+	e := echo.New()
+
+	lifejourneys.LifeJourneyService = lifejourneys.LifeJourneyServiceStruct{Filename: "../life_journeys.json"}
+	// Setup http request using httptest
+	req := httptest.NewRequest(http.MethodGet, "/lifejourneys/1/benefits", nil)
+	// Create a httptest record
+	rec := httptest.NewRecorder()
+	// Create a new Echo Context
+	c := e.NewContext(req, rec)
+	c.SetParamNames("id")
+	c.SetParamValues("1")
```

Comment:
I had to set params manually for tests, I'm not sure if this is a good practice but it was the only way I could find